src/main/java/org/reactome/orthoinference/EventsInferrer.java

```diff
@@ -122,11 +116,10 @@ public static void inferEvents(Properties props, String species) throws Exceptio
 		// Set static variables (DB/Species Instances, mapping files) that will be repeatedly used
 		setInstanceEdits(personId);
 		try {
-			Map<String,String[]> homologueMappings = readHomologueMappingFile(species, "hsap", pathToOrthopairs);
-			ProteinCountUtility.setHomologueMappingFile(homologueMappings);
-			EWASInferrer.setHomologueMappingFile(homologueMappings);
-		} catch (FileNotFoundException e) {
-			logger.fatal("Unable to locate " + speciesName +" mapping file: hsap_" + species + "_mapping.txt. Orthology prediction not possible.");
+			readAndSetHomologueMappingFile(species, "hsap", pathToOrthopairs);
+			readAndSetGeneNameMappingFile(species, pathToOrthopairs);
+		} catch (Exception e) {
+			logger.fatal("Unable to locate " + speciesName +" mapping file: hsap_" + species + "_mapping.tsv. Orthology prediction not possible.");
```

Comment:
The exception object can be added as the second argument to the `logger.fatal` method.

Reply:
I decided to just have it print the stack trace underneath this log statement